frontend/src/app/analytics/Analytics.tsx

```diff
@@ -147,32 +215,21 @@ export const Analytics = () => {
             <div className="grid-col-3">
               <div className="card display-flex flex-column flex-align-center">
                 <h2>Negative tests</h2>
-                <h1>26</h1>
+                <h1>{negativeTests}</h1>
                 <p></p>
               </div>
             </div>
             <div className="grid-col-3">
               <div className="card display-flex flex-column flex-align-center">
                 <h2>Positivity rate</h2>
```

Comment:
Can you add a caption for this one explaining that the positivity rate is positive tests/total tests?